README.md

```diff
@@ -141,6 +141,22 @@ validate_data_spec(ok_data, GreaterThanSpec) # raise Exception
 "field: XXX not well-formatted"
 ```
 
+### Spec Object Wrapper
+
+- Inherit Spec wrapper, the spec object can simply valdates data with a class method. (see [test_wrapped_spec_class_validate](https://github.com/travisliu/data-spec-validator/blob/feat/spec-object-wrapper/test/test_spec.py#L993) for more)
```

Comment:
s/valdates/validate/g

Reply:
that's embarrassing. The typo has been corrected!